docs/content/reference/cli/authelia/authelia_storage_bans_user_list.md

```
@@ -0,0 +1,70 @@
+      --postgres.schema string                 the PostgreSQL schema name (default "public")
+      --postgres.ssl.certificate string        the PostgreSQL ssl certificate file location
+      --postgres.ssl.key string                the PostgreSQL ssl key file location
+      --postgres.ssl.mode string               the PostgreSQL ssl mode (default "disable")
+      --postgres.ssl.root_certificate string   the PostgreSQL ssl root certificate file location
+      --postgres.username string               the PostgreSQL username (default "authelia")
+      --sqlite.path string                     the SQLite database path
+```
+
+### SEE ALSO
+
+* [authelia storage bans user](authelia_storage_bans_user.md)	 - Manages user bans
```

Markdownlint

Column: 62
Hard tabs

(MD010, no-hard-tabs)